content/docs/building-apps/connect-to-anchors/setup-for-anchored-assets.mdx

```diff
@@ -349,7 +349,7 @@ Once we’ve found an Anchor that supports deposit and withdrawal, we can begin
 
 [1]: https://github.com/stellar/docs-wallet/tree/trustline
 [2]: https://www.stellarx.com/.well-known/stellar.toml
-[3]: https://www.anchorusd.com/.well-known/stellar.toml
+[3]: https://stablecoin.anchorusd.com/.well-known/stellar.toml
```

Comment:
Given the updated link, would you mind also updating the TOML file [above](https://github.com/stellar/new-docs/pull/418/files#diff-6d15707eed236ce95633b1176074c4110e744c816b8952117cbc7d93f6c863d2R305-R344) that it references?

Reply:
I updated the StellarX one, too.